src/ark/utils/data_utils.py

```diff
@@ -792,3 +805,199 @@ def stitch_images_by_shape(data_dir, stitched_dir, img_sub_folder=None, channels
         current_img = stitched_data.loc['stitched_image', :, :, chan].values
         image_utils.save_image(os.path.join(stitched_subdir, chan + '_stitched' + file_ext),
                                current_img)
+
+
+@delayed
+def _convert_ct_fov_to_adata(fov_dd: dd.DataFrame, var_names: list[str], obs_names: list[str], save_dir=None) -> str:
+    """Converts the cell table for a single FOV to an `AnnData` object and saves it to disk as a
+    `Zarr` store.
+
+    Parameters
+    ----------
+    fov_dd : dd.DataFrame
+        The cell table subset on a single FOV.
+    var_names: list[str]
+        The marker names to extract from the cell table.
+    obs_names: list[str]
+        The cell-level measurements and properties to extract from the cell table.
+    save_dir: str | os.PathLike
+        The directory to save the `AnnData` object to.
+
+    Returns
+    -------
+    str
+        The path of the saved `AnnData` object.
+    """
+    
+    fov_dd: dd.DataFrame = fov_dd.sort_values(by=settings.CELL_LABEL, key=ns.natsort_key).reset_index()
+    fov_id: str = fov_dd[settings.FOV_ID].iloc[0]
+
+    # Set the index to be the FOV and the segmentation label to create a unique index
+    fov_dd.index = list(map(lambda label: f"{fov_id}_{int(label)}", fov_dd[settings.CELL_LABEL]))
+
+    # Extract the X matrix
+    X_dd: dd.DataFrame = fov_dd[var_names]
+    
+    # Extract the obs dataframe and convert the cell label to integer
+    obs_dd: dd.DataFrame = fov_dd[obs_names].astype({settings.CELL_LABEL: int, settings.FOV_ID: str})
+    obs_dd["cell_meta_cluster"] = pd.Categorical(obs_dd["cell_meta_cluster"].astype(str))
+
+    # Move centroids from obs to obsm["spatial"]
+    obsm_dd = obs_dd[[settings.CENTROID_0, settings.CENTROID_1]].rename(columns={settings.CENTROID_0: "centroid_x", settings.CENTROID_1: "centroid_y"})
```

Comment:
Technically region props places (0,0) in the upper left corner of the image, with centroid-0 being distance along the y-axis (downwards) and centroid-1 being a regular x-axis position.  It's a very annoying and minor issue, but wondering your thoughts on the best way to make this fact clear to users.

Reply:
Just added to the centroid docs about the origin of NumPy Arrays in the `data_types.md` file. Let me know your thoughts!

Reply:
The documentation explains it well!